frontend/src/components/dialogs/SaveLinkDialog.tsx

```
@@ -0,0 +1,507 @@
+                </span>
+              </div>
+              
+              <div className="grid grid-cols-3 gap-2">
+                {folders?.slice(0, 6).map((folder) => {
+                  const isActive = selectedFolderId === folder.memberFolderId;
+                  return (
+                    <div
+                      key={folder.memberFolderId}
+                      onClick={() => setSelectedFolderId(isActive ? null : folder.memberFolderId)}
+                      className={`${styles.folderTile} ${isActive ? styles.folderTileActive : ''} group`}
+                    >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)